pom.xml

```diff
@@ -131,35 +124,23 @@
           <groupId>org.springframework.boot</groupId>
           <artifactId>spring-boot-maven-plugin</artifactId>
           <version>${spring-boot.version}</version>
-          <configuration>
-            <classifier>exec</classifier>
-          </configuration>
-          <executions>
-            <execution>
-              <goals>
-                <goal>repackage</goal>
-              </goals>
-            </execution>
-          </executions>
-        </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-failsafe-plugin</artifactId>
-          <executions>
-            <execution>
-              <goals>
-                <goal>integration-test</goal>
-                <goal>verify</goal>
-              </goals>
-            </execution>
-          </executions>
         </plugin>
       </plugins>
     </pluginManagement>
     <plugins>
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
```

Comment:
I'm not a Maven expert, but shouldn't the plugin configuration be in pluginManagement and only have the plugin activation in plugins?

Reply:
Not sure either but I think `pluginManagement` is only used for defining the version of the plugin since you could have several execution of the same plugin with different configuration in the `plugins` section…

Reply:
We should probably get this sorted out, since I doubt anyone wants to go through another round of cleaning the POMs

Reply:
There is no strict rule as far as I know. You just define the common things in the pluginManagement as well as in the dependencyManagement and they get inherited and if needed extended or overwritten. My thinking when leaving only the versions here was the same as @metacosm. But to be fair all this time, I've been having seconds thoughts why do we need pluginManagement and dependencyManagement in the single module projects at all.

Reply:
I'm fine with whatever you want to do. It's a minor issue anyway...

Reply:
@metacosm any thoughts?

Reply:
I tend to agree that the management parts seem to be overkill for boosters… but since that's how things are, unless you want to spend more time changing all the boosters, I'd say let's keep things as they are…

Reply:
I'm changing them all anyway, so it's not gonna make much more extra work

Reply:
In my opinion boosters being the example applications, should be as polished as possible. This isn't as important for other projects.

Reply:
Although, if we plan to get rid of the booster-parent, we might have to reintroduce pluginManagement, as at the moment booster-parent redefines a lot of versions there.

Reply:
Indeed. That has to be taken into consideration…

Reply:
Well we do plan to remove the booster parent don't we? If that's the case, I say we don't do the extra work now just to have to reintroduce the missing sections later

Reply:
Agreed.

Reply:
Agreed.